packages/player-component/index.js

```diff
@@ -320,8 +320,13 @@ class Player extends Nanocomponent {
   }
 
   renderPlayer () {
+    const isAuthenticated = !!this.state.user.uid
+
     return html`
       <div class="player-component flex flex-column h-100">
+        ${!isAuthenticated
+          ? html`<span class="flex flex-row w-100 justify-center bb b--light-silver">Log in to listen to full song</span>`
```

Comment:
I'm not sure if this border bottom color is correct

Reply:
You can use the following classes list instead
```b--mid-gray b--mid-gray--light b--near-black--dark```

Reply:
```suggestion
          ? html`<span class="flex flex-row w-100 justify-center bb b--mid-gray">Log in to listen to full song</span>`
```

Of the three options @auggod suggested above, `b--mid-gray` seems to match the UI in the controls below the best. Doesn't seem to make much a difference in dark mode, but in light mode note the differences between the top and bottom horizontal outlines on the second and third screenshots. Here is how they're all showing up for me:
### `b--mid-gray`
<img width="1680" alt="b--mid-gray" src="https://user-images.githubusercontent.com/60944077/149664762-658c664a-e3fc-47d2-b5f9-7010d8fa8e93.png">

### `b--mid-gray--light`
<img width="1680" alt="b--mid-gray--light" src="https://user-images.githubusercontent.com/60944077/149664764-a1d5fb4f-2df4-4105-98d0-f81845e02b9b.png">

### `b--near-black--dark`
<img width="1680" alt="b--near-black--dark" src="https://user-images.githubusercontent.com/60944077/149664768-1bfb8e2c-cf97-4af8-8399-7c7480f034e1.png">

Reply:
The 'b--mid-gray' (the first option above) seems to make the most sense in the UI to my eyes. Once that change is made this should be ready to merge.